src/components/Layout/Navbar.js

```diff
@@ -25,13 +26,23 @@ class Navbar extends React.Component {
     isOpen: false,
   }
   render() {
+    let page = this.props.gaLabel
```

Comment:
I tried to get it like it's described in the page - but I couldn't make it work in production, because there is no window object/href property of the location prop during build time :thinking: 
+ here gaLabel (short for google analytics label) is a 'short' version of the page as per google sheet made by Luca (ex. 'landing page' is 'lp', locations is 'lo' etc.). I can't think of a better way to do this than to pass every time the label as a prop :thinking: